### DISCUSSION TITLE
Insert the Tags on Projects and Activities.

### DISCUSSION THREAD

**Discussion by WhiteTiger-IT** [kimai/kimai#2305] [Questions & Answers]
Is there no way to use the tags in the other tabs too?
They can be used to categorize customers, projects or even individual activities.

**Comment by kevinpapst** [comment]
Nope, not yet

**Comment by WhiteTiger-IT** [comment]
Are you planning to introduce them in an upcoming version?
Tags and Categories are two features that Kimani lacks.
Maybe you can handle them in two steps: first introduce categories and tags. At a later time you can insert a report.
In the meantime, by using them, we can all understand which reports we need and which we do not have great needs.
